### ISSUE TITLE
How to enable proper error logging (e.g. DB table does not exist, incompatible event adapter, etc)

### ISSUE THREAD

**Issue by ning-tan** [dnvriend/akka-persistence-jdbc#229]
Hello, I'm new to Scala, Akka, persistent actors, and this library, so I apologize if my question seems silly.

I'm trying to set up an Oracle backing store for a minimal POC using the official Akka persistence sample project found here: https://developer.lightbend.com/start/?group=akka&project=akka-samples-persistence-scala. 

Using the OracleProfile and the oracle-application.conf provided by your library, I finally was able to get the POC to work. However, along the way I encountered setup issues such as incorrect table names, etc, and the sample application logged 0 errors! I had to use the debugger and browse my way through the various call stacks to stumble upon the correct clue: ORA-00942: table or view does not exist.

Before I figured this part out through the debugger, the application was starting and finishing without the expected output, but also without any error entries in the log.

I'm pretty sure I'm missing something very simple. I would have expected an Oracle error to be in my face in the log.

Can you show me how to configure logging so that these errors show up? Thank you very much.

BTW, below are the log-related settings I tried to use:

akka {
  stdout-loglevel = DEBUG // defaults to WARNING can be disabled with off. The stdout-loglevel is only in effect during system startup and shutdown
  log-dead-letters-during-shutdown = on
  loglevel = debug
  log-dead-letters = on
  log-config-on-start = on // Log the complete configuration at INFO level when the actor system is started

  loggers = ["akka.event.slf4j.Slf4jLogger"]
  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"

  actor {
    debug {
      receive = on // log all messages sent to an actor if that actors receive method is a LoggingReceive
      autoreceive = on // log all special messages like Kill, PoisoffPill etc sent to all actors
      lifecycle = on // log all actor lifecycle events of all actors
      fsm = on // enable logging of all events, transitioffs and timers of FSM Actors that extend LoggingFSM
      event-stream = on // enable logging of subscriptions (subscribe/unsubscribe) on the ActorSystem.eventStream
      unhandled = on
    }
  }
}

logger.scala.slick=DEBUG

**Comment by WellingR** [comment]
you have enabled the "akka.event.slf4j.Slf4jLogger"  which is the first step, in addition you also need to configure a logging backend.

What you can do is add a dependency to [logback-core](https://mvnrepository.com/artifact/ch.qos.logback/logback-core/1.2.3)

And place a basic `logback.xml` file in your `src/main/resources` directory

```xml
<configuration>

  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
    <!-- encoders are assigned the type
         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
    <encoder>
      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
    </encoder>
  </appender>

  <root level="debug">
    <appender-ref ref="STDOUT" />
  </root>
</configuration>
```

see https://logback.qos.ch/manual/configuration.html for more details

**Comment by ning-tan** [comment]
Thanks a lot for the suggestion. I'll implement the above and report back. I'm familiar with logback.

**Comment by ning-tan** [comment]
@WellingR thank you very much for the logback tips. I got logging working. I had to make a minor tweak. Instead of adding a dependency to logback-core, I needed logback-classic. Otherwise I got these warnings in stdout (and no debug logging):

SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
SLF4J: Failed to load class "org.slf4j.impl.StaticMDCBinder".
SLF4J: Defaulting to no-operation MDCAdapter implementation.
SLF4J: See http://www.slf4j.org/codes.html#no_static_mdc_binder for further details.
